src/components/MetricCard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+const cardStyle = makeStyles(theme => ({
+  root: {
+    padding: theme.spacing(1, 0),
+    textAlign: 'center'
+  },
+  title: {
+    borderTop: '1px solid',
+    padding: theme.spacing(1, 1)
+  },
+  body: {
+    color: '#5080AD',
+    padding: theme.spacing(4, 1)
+  }
+}));
+
+const metricCard = props => {
```

Comment:
Same thing as in #10:

1. React components have a capital-case convention
2. Object destructuring as arguments is your friend!